--- herrie-1.2/GNUmakefile~	2007-01-01 16:28:38.000000000 +0100
+++ herrie-1.2/GNUmakefile	2007-01-02 00:26:38.000000000 +0100
@@ -26,7 +26,7 @@
 
 VERSION=	1.2
 
-PREFIX?=	/usr/local
+PREFIX?=	/usr
 OS:=		$(shell uname)
 MANDIR=		$(PREFIX)/share/man
 TRANSDIR=	$(PREFIX)/share/locale
@@ -118,6 +118,8 @@
 endif
 endif
 
+CFLAGS+=       $(OPTFLAGS)
+
 # libao dies with -Wstrict-prototypes - fixed in trunk
 CFLAGS+=	$(shell pkg-config --cflags glib-2.0 gthread-2.0) \
 		-I$(PREFIX)/include -DAUDIO_OUTPUT=\"$(AUDIO_OUTPUT)\" \
@@ -136,13 +138,13 @@
 all: herrie$(BINARY_EXT) herrie.1 $(LOCALES:%=%.mo)
 
 install: all
-	install -s herrie$(BINARY_EXT) $(PREFIX)/bin/herrie$(BINARY_EXT)
-	install -m 444 herrie.1 $(MANDIR)/man1/herrie.1
+	install -s herrie$(BINARY_EXT) $(DESTDIR)$(PREFIX)/bin/herrie$(BINARY_EXT)
+	install -m 444 herrie.1 $(DESTDIR)$(MANDIR)/man1/herrie.1
 	$(foreach lang,$(LOCALES), \
-		mkdir -p $(TRANSDIR)/$(lang)/LC_MESSAGES)
+		mkdir -p $(DESTDIR)$(TRANSDIR)/$(lang)/LC_MESSAGES)
 	$(foreach lang,$(LOCALES), \
 		install -m 444 $(lang).mo \
-		    $(TRANSDIR)/$(lang)/LC_MESSAGES/herrie.mo)
+		    $(DESTDIR)$(TRANSDIR)/$(lang)/LC_MESSAGES/herrie.mo)
 
 clean:
 	rm -f herrie$(BINARY_EXT) *.o *.mo *.core
