--- herrie-1.3/GNUmakefile~	2007-01-22 23:40:35.000000000 +0100
+++ herrie-1.3/GNUmakefile	2007-01-22 23:43:39.000000000 +0100
@@ -137,6 +137,8 @@
 endif
 endif
 
+CFLAGS+=	$(OPTFLAGS)
+
 OBJS+=		audio_file.o audio_output_$(AUDIO_OUTPUT).o config.o \
 		gui_browser.o gui_draw.o gui_input.o gui_msgbar.o \
 		gui_playq.o gui_vfslist.o main.o playq.o vfs.o \
@@ -154,13 +156,13 @@
 all: $(APP_NAME) $(APP_NAME).1.gz $(LOCALES:%=%.mo)
 
 install: all
-	install -s $(APP_NAME) $(PREFIX)/bin/$(APP_NAME)$(BINARY_EXT)
-	install -m 444 $(APP_NAME).1.gz $(MANDIR)/man1/
+	install -s $(APP_NAME) $(DESTDIR)$(PREFIX)/bin/$(APP_NAME)$(BINARY_EXT)
+	install -m 444 $(APP_NAME).1.gz $(DESTDIR)$(MANDIR)/man1/
 	$(foreach lang,$(LOCALES), \
-		mkdir -p $(TRANSDIR)/$(lang)/LC_MESSAGES)
+		mkdir -p $(DESTDIR)$(TRANSDIR)/$(lang)/LC_MESSAGES)
 	$(foreach lang,$(LOCALES), \
 		install -m 444 $(lang).mo \
-		    $(TRANSDIR)/$(lang)/LC_MESSAGES/$(APP_NAME).mo)
+		    $(DESTDIR)$(TRANSDIR)/$(lang)/LC_MESSAGES/$(APP_NAME).mo)
 
 clean:
 	rm -f *.core *.mo *.o .gdb_history
